jupyter_server/extension/manager.py

```diff
@@ -355,6 +355,8 @@ def link_extension(self, name, serverapp):
                 self.linked_extensions[name] = True
                 self.log.info("{name} | extension was successfully linked.".format(name=name))
             except Exception as e:
+                if serverapp.reraise_server_extension_failures:
```

Comment:
There's a logged exception above on L345 that should probably get included in this since `add_extension()` is part of the `init_server_extensions()` call sequence.

Reply:
Done